cursorless-talon/src/modifiers/modifiers.py

```diff
@@ -27,7 +27,6 @@ def cursorless_simple_modifier(m) -> dict[str, str]:
     "<user.cursorless_simple_scope_modifier>",  # funk, state, class, every funk
     "<user.cursorless_ordinal_scope>",  # first past second word
     "<user.cursorless_relative_scope>",  # next funk, 3 funks
-    "<user.cursorless_surrounding_pair_force_direction>",  # DEPRECATED "left quad" / "right quad"
```

Comment:
This also added to the compilation time and I didn't want to create a second flattened list just for something that's deprecated and no one is using afaik.